cmd/kubelet/app/server.go

```diff
@@ -166,6 +166,8 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 				glog.Fatal(err)
 			}
 
+			initForOS()
```

Comment:
Can we call this inside `Run`? `cmd.Run` should only really be loading configuration and passing it to `Run`.

Reply:
I think we can, I will update it.
It doesn't really matter where that it is called, it should be only after the flags are parsed. It just checks if the ```--register-service``` or ```--unregister-flag``` flag was passed in which case it will only create/remove the service and exit. If those flags were not passed, nothing happens.